stable/xray/README.md

```diff
@@ -200,7 +205,7 @@ export POSTGRESQL_PASSWORD=password2_X
 export POSTGRESQL_DATABASE=xraydb
 
 export XRAY_POSTGRESQL_CONN_URL="postgres://${POSTGRESQL_HOST}:${POSTGRESQL_PORT}/${POSTGRESQL_DATABASE}?sslmode=disable"
-helm install -n xray \
+helm upgrade --install -n xray \
```

Comment:
@shahiinn  Pls use —namespace instead of -n, so the same command works for both v2 and v3 helm